playground/theme-builder/package.json

```diff
@@ -1,15 +1,17 @@
 {
-  "name": "ui-example",
+  "name": "theme-builder",
   "version": "0.1.0",
   "private": true,
   "scripts": {
     "build": "next build",
     "dev": "next dev",
     "format": "npx prettier . --write",
     "lint": "next lint",
-    "start": "next start"
+    "start": "next start",
+    "prepare": "next telemetry disable"
   },
   "dependencies": {
+    "@clerk/ui": "^0.1.0",
```

Comment:
This threw me off initially because I'm used to `workspace:*` in PNPM; but I think this is the right way to do it (just match our local version)